examples/nidaqmx_analog_input/pyproject.toml

```
@@ -6,14 +6,15 @@ authors = ["National Instruments"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-nidaqmx = ">=0.6.3"
+nidaqmx = { version = ">=0.8.0-dev0", allow-prereleases = true }
```

While working on https://github.com/ni/nidaqmx-python/pull/338 I realized this is missing `nidaqmx`'s `grpc` extra.
```suggestion
nidaqmx = { version = ">=0.8.0-dev0", extras = ["grpc"], allow-prereleases = true }
```

---

Ok. Updated.

---

Adding the extras to the dev dependency seems to work.

---

Thanks for the info. It is working as expected. 👍